### ISSUE TITLE
Roberta- learning rate

### ISSUE THREAD

**Issue by inbalcroitoru** [magud/fake-news-detection#1]
hi, I saw you wrote that roberta preformed best and that lr is the most importent hyperparameter, which one of the options was the best fit?
